tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
-envlist = py27, py35, py36, py37, py38, pypy
+envlist = py27, py35, py36, py37, py38, pypy, flake8
```

Comment:
It may be a good opportunity to change this list to `flake8, py`, so that by default `tox` runs Flake8 and the tests just for the Python version that the user has installed.

Reply:
I changed it by: `flake8, py2, py3` as we don't have deprecated the Python 2 support (yet).